extra-wm/i3status-rust/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=i3status-rust
+PKGSEC=x11
+PKGDEP="pulseaudio"
+BUILDDEP="rustc"
+PKGDES="A Rust re-implementation of i3status"
```

Comment:
Please append the following...

```
ABTYPE=rust
USECLANG=1
```

To make use of Autobuild3’s standard build routine for Rust sources. `USECLANG=1` is used here for Rust LTO.